pom.xml

```diff
@@ -13,7 +13,7 @@
     <!-- Must be duplicated for versions:set plugin to work in builds -->
     <groupId>com.ibm</groupId>
     <artifactId>sol005-lifecycle-driver</artifactId>
-    <version>0.2.1</version>
+    <version>0.2.2-SNAPSHOT</version>
```

Comment:
You might need to append RC1 instead of SNAPSHOT as per the latest release process document.